hbase-common/src/main/java/org/apache/hadoop/hbase/HConstants.java

```diff
@@ -158,6 +158,12 @@
   public static final String HBASE_MASTER_NORMALIZER_CLASS =
     "hbase.master.normalizer.class";
 
+  /** Config for min age of region before being considerded for merge in mormalizer */
+  public static final int DEFAULT_MIN_DAYS_BEFORE_MERGE = 3;
+
+  public static final String HBASE_MASTER_DAYS_BEFORE_MERGE =
+      "hbase.master.normalize.daysBeforeMerge";
+
```

Comment:
These defines are used in one place only. They don't need to be in this global file of defines. Only stuff that is used in multple places should go here. FYI.

Reply:
Agreed. So maybe i can move it to hbase-defaults.xml and read it from there.

Reply:
I think we can move both these constants to MergeNormalizer. If we move to hbase-default, we don't need this constant `DEFAULT_MIN_DAYS_BEFORE_MERGE` because `hbase.master.normalize.daysBeforeMerge` will always have value, either from hbase-default or hbase-site

Reply:
If particular to an implemenation, yeah, move to the implementation.